Resources/views/Bootstrap/grid.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '::base.html.twig' %}
+
+{% block header_title %}
+<h2>Grid</h2>
+{% endblock %}
+
+{% block main %}
+<div class="block-link">
+  <p> Meer info en voorbeelden op <a href="http://getbootstrap.com/css/#grid">http://getbootstrap.com/css/#grid</a></p>
```

Comment:
ik zou hier trouwens ook een voorbeeld grid insteken. Het is wat raar dat er op iedere pagina voorbeelden staan, en hier enkel een link naar de site van bootstrap.